server/mdm/apple/apple_mdm.go

```diff
@@ -397,12 +384,80 @@ func NewDEPService(
 // processDeviceResponse processes the device response from the device sync
 // DEP API endpoints and assigns the profile UUID associated with the DEP
 // client DEP name.
-func (d *DEPService) processDeviceResponse(ctx context.Context, depClient *godep.Client, resp *godep.DeviceResponse, tmID *uint) error {
+func (d *DEPService) processDeviceResponse(ctx context.Context, depClient *godep.Client, resp *godep.DeviceResponse) error {
 	if len(resp.Devices) < 1 {
 		// no devices means we can't assign anything
 		return nil
 	}
 
+	var addedDevices []godep.Device
+	var deletedSerials []string
+	var modifiedDevices []godep.Device
+	var modifiedSerials []string
+	for _, device := range resp.Devices {
+		level.Debug(d.logger).Log(
+			"msg", "device",
+			"serial_number", device.SerialNumber,
+			"device_assigned_by", device.DeviceAssignedBy,
+			"device_assigned_date", device.DeviceAssignedDate,
+			"op_date", device.OpDate,
+			"op_type", device.OpType,
+			"profile_assign_time", device.ProfileAssignTime,
+			"push_push_time", device.ProfilePushTime,
+			"profile_uuid", device.ProfileUUID,
+		)
```

Comment:
:+1:

Reply:
I think this is your code! great idea to log this info +1

Reply:
:D I don't remember adding this, it might come from nanomdm then!